src/components/AddItem.js

```diff
@@ -29,8 +29,10 @@ const AddItem = () => {
   const addToDatabase = (e, inputValue, setInputValue) => {
     const newItem = {
       name: inputValue,
-      lastPurchased: null,
+      recentPurchase: null,
       howSoon: e.target['how-soon'].value,
+      previousPurchase: null,
```

Comment:
How are you using `previousPurchase` and `recentPurchase`? You can satisfy the AC by having only 1 data point that represents the date of the most recent purchase, but maybe you have another reason for keeping the other date around?

Reply:
We added `previousPurchase` to test `calculateEstimate` 
but it's true we don't need to store it in DB
will remove it now:))